src/dataflow/src/source/kinesis.rs

```diff
@@ -37,126 +41,237 @@ lazy_static! {
     .expect("Can construct an intgauge for millis_behind_latest");
 }
 
-lazy_static! {
```

Comment:
Why are we getting rid of this?

Reply:
It's part of a global metric mz_bytes_read in source/mod.rs